src/model/subnet.ts

```diff
@@ -140,6 +140,26 @@ export type NeuronRegCostHistoryResponse = {
 	data: NeuronRegCostHistory[];
 };
 
+export type NeuronDeregistration = {
+	height: bigint;
```

Comment:
```suggestion
	height: number;
```

height is of type `number`, not `bigint`